src/docker/README.md

```diff
@@ -19,17 +19,16 @@ This Docker compose is designed to work with an end-to-end testing environment,
 
 There are a few steps to start up the local end-to-end testing environment:
 
-**1. Configure the processor.**
+**1. Set environment variables.**
 
-See the file `processor/config-template-local.yaml` for an example configuration.
-Copy and rename this file to `config.yaml`, which is ignored by git, so that the docker compose will pick it up.
+See the file `example.env` for an example configuration.
+Copy and rename this file to `.env`, which is ignored by git, so that the docker compose will pick it up.
```

Comment:
This is not currently ignored by git on my machine

Do you need to add a gitignore?